base/applications/rapps/configparser.cpp

```diff
@@ -28,11 +29,8 @@ static CSectionNames g_Names;
 static
 ATL::CStringW GetINIFullPath(const ATL::CStringW& FileName)
 {
-    ATL::CStringW szDir;
-    ATL::CStringW szBuffer;
-
-    GetStorageDirectory(szDir);
-    szBuffer.Format(L"%ls\\rapps\\%ls", szDir.GetString(), FileName.GetString());
+    ATL::CStringW szBuffer = CAvailableApps::m_Strings.szAppsPath;
+    szBuffer += L"\\" + FileName;
```

Comment:
What problem does this solve?

Reply:
In your previous comment you complained that I used `PathAppendW` that necessitated the explicit `MAX_PATH` limit.

Reply:
yes, but the original code worked fine?

Reply:
The original code part that was concatenating `szDir` + the hardcoded `rapps` was in effect reconstructing the already-constructed `szAppsPath`.